src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\Mime\Header\Headers;
+use Symfony\Component\Mime\Part\AbstractPart;
+use Twig\Extra\CssInliner\CssInlinerExtension;
+use Twig\Extra\Inky\InkyExtension;
+use Twig\Extra\Markdown\MarkdownExtension;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class NotificationEmail extends TemplatedEmail
+{
+    public const IMPORTANCE_URGENT = 'urgent';
+    public const IMPORTANCE_HIGH = 'high';
+    public const IMPORTANCE_MEDIUM = 'medium';
+    public const IMPORTANCE_LOW = 'low';
+
+    private const DEFAULT_HTML_TEMPLATE_PATH = '@email/notification.html.twig';
+    private const DEFAULT_TEXT_TEMPLATE_PATH = '@email/notification.txt.twig';
```

Comment:
Why not adding the default path to the vars itself? The look useless to me

Reply:
Because we are reusing them in the code